internal/consensus/state.go

```diff
@@ -1452,6 +1452,28 @@ func (cs *State) defaultDoPrevote(ctx context.Context, height int64, round int32
 		return
 	}
 
+	/*
+		Before prevoting on the block received from the proposer for the current round and height,
+		we request the Application, via `ProcessProposal` ABCI call, to confirm that the block is
```

Comment:
Nit: I know this was pre-existing, but I don't think the markup really adds much here. The names are clear, and nothing renders it anyway. YMMV.